terms-of-service.md

```
@@ -0,0 +1,202 @@
+If you have questions regarding these Terms of Service, contact us:
+
+**Email:** [contact@aossie.org](mailto:contact@aossie.org)
+
+**Discord:** [Discord](https://discord.gg/hjUhu33uAn)
+
+
+Or open an issue on [the project's GitHub repository](https://github.com/AOSSIE-Org/OrgExplorer).
+
+---
+
+*These Terms are provided for transparency around an open-source project and are not a substitute for professional legal advice.*
```

markdownlint-cli2 (0.23.2)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)